hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -45,6 +45,17 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
             </div>
 
             <div class="form-group">
+                <label for="subscriptionState" class="col-md-3 control-label">Initial state</label>
+                <div class="col-md-9">
+                    <select required class="form-control" id="subscriptionState" name="subscriptionState" ng-model="subscription.state">
+                        <option value="PENDING">PENDING</option>
```

Comment:
Yes but up until now user had no choice. Default was to activate subscription and until it was done subscription had intermediate state: pending. Right now you ask user if he wants his subscription to have intermediate state... Thats why I think that choice should be to have either SUSPENDED (do nothing until activated) or ACTIVE (PENDING until activated) state... You ask user for a desired state in my opinion...